website/docs/basics/blocks-component-in-block.md

```diff
@@ -289,37 +377,36 @@ Exactly the same way as attributes.
 
 ### I have a component name that has multiple strings in a name, will this work?
 
-By multiple strings in a name we mean that component name has multiple words, for example, `jumbotron-cta`. The problem here is that we define components with dash and attributes with camelcase. That's why you have to fix this. We have provided you with a helper. The helper makes camelcase on all your component-name strings.
+By multiple strings in a name we mean that component name has multiple words, for example, `jumbotron-cta`. The problem here is that we define components with kebab-case and attributes with camelcase.
 
-**Example:**
+The short answer is **yes** this will work.
 
-You have a component named `jumbotron-cta` and you want to use it in your block. As described on this page, you will create a `components` key in your `manifest.json` and use `jumbotron-cta`.
+### I have a component that I want to use manually
 
-```json
-{
-	"components": {
-		"jumbotron-cta": "jumbotron-cta"
-	}
-}
-```
+We created this cool way of importing attributes and loading components in your blocks or other components. However, that doesn't mean that you can't have instances where you have to load components manually. In that case, you have to provide all the attributes that you want to use manually. The attributes that you didn't provide will not be magically set from the manifest, but you can make your life easier and map it like this.
 
-But in all places in your component where you use `setAttributes` you must use [camelize helper](https://github.com/infinum/eightshift-frontend-libs/blob/develop/scripts/helpers/camelize.js)
+For example, you have a `card` component that you will use in the `featured-posts` block.
 
-```js
+`src/Blocks/custom/featured-posts/featured-posts.php`
 
-import { camelize } from '@eightshift/frontend-libs/scripts/helpers';
+```php
+// Fetch and parse manifest using custom helper.
+$cardManifest = Components::getManifest(dirname(__DIR__, 2) . '/components/card');
```

Comment:
Where is this going to be used in the example below?

Reply:
not used. I have removed it